lua/nvim-treesitter/parsers.lua

```diff
@@ -528,9 +528,22 @@ list.hcl = {
 list.markdown = {
   install_info = {
     url = "https://github.com/MDeiml/tree-sitter-markdown",
+    location = "tree-sitter-markdown/tree-sitter-markdown",
     files = { "src/parser.c", "src/scanner.cc" },
-    branch = "main",
+    branch = "split_parser",
+  },
+  maintainers = { "@MDeiml" },
+  experimental = true,
+}
+
+list.markdown_inline = {
+  install_info = {
+    url = "https://github.com/MDeiml/tree-sitter-markdown",
+    location = "tree-sitter-markdown_inline/tree-sitter-markdown-inline",
```

Comment:
@MDeiml we removed the weird prefix here https://github.com/nvim-treesitter/nvim-treesitter/pull/3048 can you also remove it here? I think it was very intuitive why it had to be `tree-sitter-markdown_inline`